lua/autorun/server/sv_pvp_movespeed.lua

```diff
@@ -77,69 +46,55 @@ local function setSpeedFromWeight( ply, totalWeight )
 
     local newRunSpeed = baseRunSpeed * multiplier
     local newWalkSpeed = baseWalkSpeed * multiplier
-    plyWraps.SetRunSpeed( ply, math.max( newRunSpeed, minRunSpeed ) )
-    plyWraps.SetWalkSpeed( ply, math.max( newWalkSpeed, minWalkSpeed ) )
+    o_SetRunSpeed( ply, math.max( newRunSpeed, minRunSpeed ) )
+    o_SetWalkSpeed( ply, math.max( newWalkSpeed, minWalkSpeed ) )
 
     local slowerThanSlowWalk = newWalkSpeed < slowWalkSpeed
     ply:SetCanWalk( not slowerThanSlowWalk ) -- Prevent +walk from letting the player move faster when overencumbered, without having to manage a third speed type
-
-    local verySlow = newWalkSpeed < walkSpeedAlert
-    local slowDueToWeight = verySlow and baseWalkSpeed >= walkSpeedAlert
-
-    if slowDueToWeight then
-        alertAboutWeightSlowness( ply )
-    end
-end
-pvpMoveSpeed.setSpeedFromWeight = setSpeedFromWeight
-
-local function isPACWeapon( weapon )
-    return string.sub( weapon:GetClass(), 1, 4 ) == "pac_"
 end
 
 local function getWeaponWeight( weapon )
-    if isPACWeapon( weapon ) then return 0 end
+    if string.sub( weapon:GetClass(), 1, 4 ) == "pac_" then return 0 end
```

Comment:
I prefer this as its own function for readability/maintainability reasons but I won't make a fuss about it.

We could also look at making a hook